Cargo.toml

```diff
@@ -32,11 +32,11 @@ prost           = { version = "0.12.3", default-features = false, features = ["p
 bytes           = { version = "1.2",  default-features = false }
 tonic           = { version = "0.10",  default-features = false, optional = true }
 serde           = { version = "1.0",  default-features = false, optional = true }
-schemars        = { version = "0.8",  optional = true }
 subtle-encoding = { version = "0.5",  default-features = false }
 base64          = { version = "0.21", default-features = false, features = ["alloc"] }
 flex-error      = { version = "0.4",  default-features = false }
 ics23           = { version = "0.11.0", default-features = false }
+pbjson          = { version = "0.6.0", default-features = false }
```

Comment:
We have some users in no-std environments, and the regular `pbjson` crate isn't cutting it there. Would you switch to the [`informalsystems-pbjson`](https://github.com/informalsystems/pbjson) instead? 
fyi: The `cosmos/ics2‍3` repo uses `informalsystems-pbjson` as well.

Reply:
Ah good point, I forgot we were using our fork for no-std support.

Reply:
Using the fork now 👍